components/mpas-albany-landice/cime_config/buildnml

```diff
@@ -229,25 +229,31 @@ def buildnml(case, caseroot, compname):
         lines.append('    <var name="layerCenterSigma"/>')
         lines.append('    <var name="layerInterfaceSigma"/>')
         lines.append('    <var name="thickness"/>')
+        lines.append('    <var name="bedTopography"/>')
         lines.append('    <var name="lowerSurface"/>')
         lines.append('    <var name="upperSurface"/>')
         lines.append('    <var name="cellMask"/>')
         lines.append('    <var name="edgeMask"/>')
-#    lines.append('    <var name="vertexMask"/>')
-#    lines.append('    <var name="normalVelocity"/>')
         lines.append('    <var name="uReconstructX"/>')
         lines.append('    <var name="uReconstructY"/>')
         lines.append('    <var name="surfaceSpeed"/>')
         lines.append('    <var name="basalSpeed"/>')
-        lines.append('    <var name="sfcMassBal"/>')
```

Comment:
Good suggestion - if MALI modifies the SMB field it was given in any way, we'd want to be able to quantify that, so having both `sfcMassBal` and `sfcMassBalApplied` would allow evaluating that.  I'll add that back in.